core/Makefile

```diff
@@ -42,7 +42,7 @@ FIRMWARE_P1_MAXSIZE = 786432
 FIRMWARE_P2_MAXSIZE = 917504
 FIRMWARE_MAXSIZE    = 1703936
 
-CFLAGS += -DSCM_REVISION='"$(shell git rev-parse HEAD | sed 's:\(..\):\\x\1:g')"'
```

Comment:
This was taken from https://github.com/trezor/trezor-firmware/blob/3976ef7261add5e513699d82412d6efe242a5fe1/legacy/firmware/Makefile#L131

Do we need to fix it there as well?

Reply:
in legacy makefile is using CFLAGS implicitly

in core, they're explicitly passed to scons via `CFLAGS="$(CFLAGS)"` which is where the `"`s get mangled